### PR TITLE
refactor ignition test

### PR DESCRIPTION
**Pull request by lalon4**

### CODE REVIEW

File: discovery-infra/tests/test_discovery_ignition_override.py [hunk 1]
```diff
@@ -61,43 +61,41 @@ def test_discovery_ignition_after_ISO_was_created(self, api_client, nodes, clust
...
         ignition_override = copy.deepcopy(TestDiscoveryIgnition.base_ignition)
         test_string = "not base 64b content"
         override_path = "/etc/test_discovery_ignition_bad_format"
         ignition_override["storage"]["files"][0]["path"] = override_path
         ignition_override["storage"]["files"][0]["contents"] = {"source": f"data:text/plain;base64,{test_string}"}
-
         try:
-            api_client.patch_cluster_discovery_ignition(cluster_id, ignition_override)
-        # TODO: change this to BadRequest
+            api_client.patch_cluster_discovery_ignition(new_cluster.id, ignition_override)
```
Comment (nshidlin) [inline discovery-infra/tests/test_discovery_ignition_override.py:98]: Why not add this to cluster?

Comment (lalon4) [reply discovery-infra/tests/test_discovery_ignition_override.py]: Added

**Pull request merged**